doc/bql.rst

```diff
@@ -136,10 +136,10 @@ For those that are modeled, it specifies their statistical type.
 
 ``GUESS SCHEMA FOR <name>``
 
-   Guess a population schema for the table *name*. A schema maps the columns in
-   *name* to heuristically guessed statistical types. Statistical types can be
-   guessed to be nominal or numerical. Columns can also be guessed to be
-   ignored.
+   Guess a population schema for the table *name*. Returns a temporary table
+   mapping each column in *name* to its heuristically guessed statistical type
+   and the heuristic reason for the guess. Statistical types can be guessed to
+   be nominal or numerical. Columns can also be guessed to be ignored.
```

Comment:
Not a 'temporary table' -- just a table, like any other `SELECT` query.  (The term 'temporary table' applies to named tables that can be persistently referenced, either permanently, with `CREATE TABLE`, or temporarily within the scope of a single database connection, with `CREATE TEMP TABLE`.)

I suggest writing the schema of the resulting table in the form of a `CREATE TABLE` statement with a dummy name.  E.g., you might say:  'The query yields a table created as if by the following `CREATE TABLE`:

```
CREATE TABLE t(
    name TEXT NOT NULL UNIQUE,
    stattype TEXT NOT NULL
)
```